Sources/FoundationNetworking/URLSession/libcurl/MultiHandle.swift

```diff
@@ -384,13 +481,12 @@ fileprivate extension URLSession._MultiHandle {
             timeoutSource = nil
             queue.async { self.timeoutTimerFired() }
         case .milliseconds(let milliseconds):
-            if (timeoutSource == nil) || timeoutSource!.milliseconds != milliseconds {
-                //TODO: Could simply change the existing timer by using DispatchSourceTimer again.
-                let block = DispatchWorkItem { [weak self] in
-                    self?.timeoutTimerFired()
-                }
-                timeoutSource = _TimeoutSource(queue: queue, milliseconds: milliseconds, handler: block)
+            //TODO: Could simply change the existing timer by using DispatchSourceTimer again.
```

Comment:
Does it has to be done or one issue will be created ?
```suggestion
            // TODO: Could simply change the existing timer by using DispatchSourceTimer again.
```

Reply:
Can't really say on behalf on this comment author😞 This was here [from the beginning](https://github.com/apple/swift-corelibs-foundation/blob/5fdd36b193cb1f4f97b4e65c0ed5588fafea30c7/Foundation/NSURLSession/MultiHandle.swift#L337). Guess it is better to leave as is to keep the diff less noisy.

Reply:
Yeah, let the Todo, I thought you've added it